ocs_ci/deployment/fusion_data_foundation.py

```diff
@@ -119,14 +121,32 @@ def setup_fdf_pre_release_deployment(self):
         out = run_cmd(cmd)
         assert "patched" in out
 
+    def verify_fdf_installation(self):
+        """
+        Verify the FDF installation was successful.
+        """
+        logger.info("Verifying FDF installation")
+        fusion_service_instance_health_check()
+        self.get_installed_version()
+        logger.info("FDF successfully installed")
 
-def verify_fdf_installation():
-    """
-    Verify the FDF installation was successful.
-    """
-    logger.info("Verifying FDF installation")
-    fusion_service_instance_health_check()
-    logger.info("FDF successfully installed")
+    def get_installed_version(self):
+        """
+        Retrieve the installed FDF version.
+
+        Returns:
+            str: Installed FDF version.
+
+        """
+        logger.info("Retrieving installed FDF version")
+        results = run_cmd(
+            f"oc describe FusionServiceInstance {constants.FDF_SERVICE_NAME} "
```

Comment:
can you try with `oc get --output yaml` if it's returning also same data?

Reply:
Fixed